ipam/allocator.go

```diff
@@ -56,7 +56,7 @@ type Allocator struct {
 	nicknames         map[mesh.PeerName]string  // so we can map nicknames for rmpeer
 	pendingAllocates  []operation               // held until we get some free space
 	pendingClaims     []operation               // held until we know who owns the space
-	pendingConsenses  []operation               // held until consensus achieved
+	pendingPrimes     []operation               // held until consensus achieved
```

Comment:
Established isn't enough - we need to have a copy of it too. Have gone with "held while our ring is empty"